wal/checkpoint.go

```diff
@@ -167,7 +168,7 @@ func Checkpoint(w *wal.WAL, from, to int, keep func(id uint64) bool, mint int64)
 		rec := r.Record()
 
 		switch dec.Type(rec) {
-		case RecordSeries:
+		case record.RecordSeries:
```

Comment:
These all stutter now as well. `record.Series`, `record.Samples`, and `record.Tombstones` would be nicer to read.